Index: b/packages/xo-server/sample.config.toml
===================================================================
--- a/packages/xo-server/sample.config.toml
+++ b/packages/xo-server/sample.config.toml
@@ -105,7 +105,7 @@
 # Port on which the server is listening on.
 #
 # Default: undefined
-port = 80
+port = 8080
 
 # Instead of `host` and `port` a path to a UNIX socket may be specified
 # (overrides `host` and `port`).
@@ -196,7 +196,7 @@ port = 80
 # Use sudo for mount with non-root user
 #
 # Default: false
-#useSudo = false
+useSudo = true
 
 #=====================================================================
 
Index: b/packages/xo-server/xo-server.service
===================================================================
--- a/packages/xo-server/xo-server.service
+++ b/packages/xo-server/xo-server.service
@@ -1,14 +1,15 @@
-# systemd service for XO-Server.
-
 [Unit]
-Description= XO Server
-After=network-online.target
+Description=XO Server
+After=network-online.target redis-server.service
 
 [Service]
-Environment="DEBUG=xo:main"
-ExecStart=/usr/local/bin/xo-server
+User=xo-server
+WorkingDirectory=/var/lib/xo-server
+ExecStart=/usr/bin/xo-server
 Restart=always
 SyslogIdentifier=xo-server
+RuntimeDirectory=xo-server
+RuntimeDirectoryMode=0750
 
 [Install]
 WantedBy=multi-user.target
Index: b/packages/xo-server/config.toml
===================================================================
--- a/packages/xo-server/config.toml
+++ b/packages/xo-server/config.toml
@@ -156,7 +156,7 @@ level = 'info'
 [logs.transport.console]
 
 [plugins]
-lookupPaths = ['./node_modules', '../', '/usr/local/lib/node_modules']
+lookupPaths = ['./node_modules', '../', '/usr/lib/xo-server/node_modules', '/usr/local/lib/node_modules']
 
 [remoteOptions]
 mountsDir = '/run/xo-server/mounts'
