Index: b/packages/xo-server/src/xo-mixins/file-restore-ng.mjs
===================================================================
--- a/packages/xo-server/src/xo-mixins/file-restore-ng.mjs
+++ b/packages/xo-server/src/xo-mixins/file-restore-ng.mjs
@@ -36,10 +36,12 @@ export default class BackupNgFileRestore
 
     // clean any LVM volumes that might have not been properly
     // unmounted
+    /*
     app.hooks.on('start', async () => {
       await Promise.all([execa('losetup', ['-D']), execa('vgchange', ['-an'])])
       await execa('pvscan', ['--cache'])
     })
+    */
 
     app.hooks.on('stop', () =>
       asyncEach(
