Index: b/packages/xo-web/src/xo-app/menu/index.js
===================================================================
--- a/packages/xo-web/src/xo-app/menu/index.js
+++ b/packages/xo-web/src/xo-app/menu/index.js
@@ -328,7 +328,7 @@ export default class Menu extends Compon
           },
         ],
       },
-      {
+      +process.env.XOA_PLAN <= 4 && {
         to: isAdmin ? 'xoa/update' : 'xoa/notifications',
         icon: 'menu-xoa',
         label: 'xoa',
@@ -437,7 +437,7 @@ export default class Menu extends Compon
           },
         ],
       },
-      isAdmin && {
+      isAdmin && +process.env.XOA_PLAN <= 4 && {
         to: '/hub/templates',
         icon: 'menu-hub',
         label: 'hubPage',
@@ -454,7 +454,7 @@ export default class Menu extends Compon
           },
         ],
       },
-      isAdmin && {
+      isAdmin && +process.env.XOA_PLAN <= 4 && {
         to: '/proxies',
         icon: 'proxy',
         label: 'proxies',
@@ -478,7 +478,7 @@ export default class Menu extends Compon
         label: 'taskMenu',
         pill: nResolvedTasks,
       },
-      isAdmin && { to: '/xosan', icon: 'menu-xosan', label: 'xosan' },
+      isAdmin && +process.env.XOA_PLAN <= 4 && { to: '/xosan', icon: 'menu-xosan', label: 'xosan' },
       !noOperatablePools && {
         to: '/import/vm',
         icon: 'menu-new-import',
@@ -558,7 +558,7 @@ export default class Menu extends Compon
               </Link>
             </li>
           )}
-          {(isAdmin || +process.env.XOA_PLAN === 5) && (
+          {(isAdmin && +process.env.XOA_PLAN <= 4) && (
             <li className='nav-item xo-menu-item'>
               <Link className='nav-link' style={{ display: 'flex' }} to='/about'>
                 {+process.env.XOA_PLAN === 5 ? (
Index: b/packages/xo-web/src/xo-app/index.js
===================================================================
--- a/packages/xo-web/src/xo-app/index.js
+++ b/packages/xo-web/src/xo-app/index.js
@@ -187,7 +187,7 @@ export const ICON_POOL_LICENSE = {
         if (getXoaPlan() === SOURCES.name) {
           poolLicenseInfoByPoolId[poolId] = {
             nHostsUnderLicense,
-            icon: () => <Icon icon='unknown-status' className='text-warning' />,
+            icon: () => {},
             nHosts,
           }
           return
Index: b/packages/xo-web/src/xo-app/home/host-item.js
===================================================================
--- a/packages/xo-web/src/xo-app/home/host-item.js
+++ b/packages/xo-web/src/xo-app/home/host-item.js
@@ -167,16 +167,6 @@ export default class HostItem extends Co
       }
 
       const proSupportStatus = this._getProSupportStatus()
-      if (proSupportStatus !== undefined && proSupportStatus.level !== 'success') {
-        alerts.push({
-          level: proSupportStatus.level,
-          render: (
-            <span>
-              {proSupportStatus.icon} {_(proSupportStatus.message)}
-            </span>
-          ),
-        })
-      }
 
       if (!host.hvmCapable) {
         alerts.push({
@@ -230,9 +220,6 @@ export default class HostItem extends Co
                 &nbsp;
                 <BulkIcons alerts={this._getAlerts()} />
                 &nbsp;
-                {proSupportStatus?.level === 'success' && (
-                  <Tooltip content={_(proSupportStatus.message)}>{proSupportStatus.icon}</Tooltip>
-                )}
               </EllipsisContainer>
             </Col>
             <Col mediumSize={3} className='hidden-lg-down'>
Index: b/packages/xo-web/src/xo-app/home/pool-item.js
===================================================================
--- a/packages/xo-web/src/xo-app/home/pool-item.js
+++ b/packages/xo-web/src/xo-app/home/pool-item.js
@@ -107,20 +107,6 @@ export default class PoolItem extends Co
     (isAdmin, poolLicenseInfo) => {
       const alerts = []
 
-      if (isAdmin && this._isXcpngPool()) {
-        const { icon, supportLevel } = poolLicenseInfo
-        if (supportLevel !== 'total') {
-          const level = supportLevel === 'partial' || getXoaPlan() === SOURCES ? 'warning' : 'danger'
-          alerts.push({
-            level,
-            render: (
-              <p>
-                {icon()} {this._getPoolLicenseIconTooltip()}
-              </p>
-            ),
-          })
-        }
-      }
       return alerts
     }
   )
